app/models/device.rb

```diff
@@ -1,3 +1,11 @@
 class Device < ApplicationRecord
   has_many :visits
+
+  def self.breakdown_by_date_range(start_date, end_date)
```

Comment:
Should this use `start_date` and `end_date`?

Reply:
Good point, thanks!